source/vscode.rst

```diff
@@ -79,7 +79,49 @@ Linux
 
 Microsoft provide Visual Studio Code packages for a number of Linux
 distributions. See the `Visual Studio Code Linux installation webpage
-https://code.visualstudio.com/docs/setup/linux`_.
+<https://code.visualstudio.com/docs/setup/linux>`__.
 
-Installing necessary extensions
--------------------------------
\ No newline at end of file
+In particular, for Debian and Ubuntu based distributions, the easiest way to install Visual Studio Code is to download and install the ``.deb`` package (64-bit) from `here <https://code.visualstudio.com/Download>`__, and then through the command line in terminal with:
+
+.. code-block:: console
+
+    $ sudo dpkg -i <file>.deb
+
+Installing the ``.deb`` package will automatically install the apt repository and signing key to enable auto-updating of the software using the system's package manager. 
+
+Customizing Visual Studio Code
+------------------------------
+
+Adding Extensions
+~~~~~~~~~~~~~~~~~
+
+There are a number of Visual Studio Code `extensions <https://marketplace.visualstudio.com/>`__ specific to certain programming languages, debuggers, and tools such as a Git repositoy controls support your code development. Please `see this webpage <https://code.visualstudio.com/docs/editor/extension-gallery>`__ and `this one <https://code.visualstudio.com/docs/introvideos/extend>`__ (also has a video tutorial) for more information and howto's. 
+
+In particular, we recommend the installaation of the following extensions:
+
+* `Liveshare <https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare-pack>`__ (Real-time collaborative coding)
+* `GitLens <https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens>`__ (Seamlessly use Git and control Git repositories within the IDE)
```

Comment:
I think this isn't needed. Visual Studio Code has inbuilt git support.

Reply:
Yes, but Gitlens significantly improves on the inbuilt Git support. I have qualified this now.